src/utils.jl

```diff
@@ -183,30 +183,44 @@ var_from_nested_derivative(x, i=0) = (missing, missing)
 var_from_nested_derivative(x::Term,i=0) = operation(x) isa Differential ? var_from_nested_derivative(arguments(x)[1], i + 1) : (x, i)
 var_from_nested_derivative(x::Sym,i=0) = (x, i)
 
-function degree(p::Sym, sym::Union{Sym,Nothing}=nothing)
+function degree(p::Sym, sym=nothing)
     if sym === nothing
         return 1
     else
         return p.name == sym.name ? 1 : 0
```

Comment:
Why not `isequal(p, sym)`?

Reply:
Did not know this function existed, is it preferred to `==` in this case?

Reply:
They behave differently. Try them in the Julia REPL.